docs/src/pages/de/installation.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Installation
+description: So installieren Sie Astro mit NPM, PNPM oder Yarn.
```

Comment:
```suggestion
description: So installierst du Astro mit NPM, PNPM oder Yarn.
```

Reply:
Fine but we need to continually either use "Du" or "du" ... a mix of both is simply wrong